ConsoleCalculator/Operations.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleCalculator
+{
+    public class Operations
+    {
+        public string firstNumber = "";
+        public string secondNumber = "";
+        public string output = "0";
+        public char previousOperator = '?';
+        public string AddDigitExceptZero(char digit)
+        {
+            if (previousOperator == '?')
+            {
+                if (firstNumber == "")
+                    firstNumber = digit.ToString();
+                else
+                    firstNumber += digit;
+                output = firstNumber;
+            }
+            else
+            {
+                if (secondNumber == "")
+                    secondNumber = digit.ToString();
+                else
+                    secondNumber += digit;
+                output = secondNumber;
+            }
+            return output;
+        }
+        public string AddZero()
+        {
+            if (previousOperator != '?' && secondNumber == "")
+            {
+                output = "0";
+                secondNumber = output;
+            }
+            else if (secondNumber == "" && firstNumber == "")
+            {
+                output = "0";
+            }
+            else if (firstNumber != "" && secondNumber == "")
+            {
+                firstNumber = firstNumber + "0";
+                output = firstNumber;
+            }
+            else if (secondNumber != "")
+            {
+                secondNumber += "0";
+                output = secondNumber;
+            }
+
+            return output;
+        }
+        public string Operator(char key)
+        {
+            switch (key)
+            {
+                case '+':
+                    return Addition();
+                case '-':
+                    return Subtraction();
+                case 'x':
```

Comment:
why not *?

Reply:
maam in the qus for multiply 'X' and 'x' this were given 
if u say then i will make this case also